### PR TITLE
First round of changes to Craft 3 documentation

### PR DESCRIPTION
**Pull request by ryanirelan**

### CODE REVIEW

**Comment by brandonkelly** [comment]
Boom!!

File: en/the-pieces-of-craft.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Structure
+* Single
+
+To review the specifics of each type, check out the [Sections and Entries](en/sections-and-entries.md) page of this documentation.
+
+## Entries
+
+Inside of the Sections are Entries. Entries are your individual pieces of content (like a cocktail recipe). A section can have one or many entries. The entries are typically inputted via the Entry Form in the Craft control panel.
+
+Each entry has a unique identifier called a `slug` that Craft uses to retrieve it and display on a web page. The `slug` is also the URI of the entry.
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Technically slugs don’t have to be unique, and they are only (possibly) a _part_ of the entry’s URI, based on the section’s Entry URI Format, which probably contains `{slug}`. The URIs are the things that must be unique, and if one is not unique, Craft will increment the slug (`slug-1`, `slug-2`…) until the resulting URI becomes unique. But if the Entry URI Format contains something else that would guarantee a unique URI (e.g. `{id}`), there’s nothing stopping every entry in the section from having the exact same slug.

I don’t think you should go into that level of detail here, but maybe just reword this to something like:

> Each entry has an identifier called a “slug”, which could be used to help give the entry a unique URL.

File: en/the-pieces-of-craft.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Each entry can only belong to one Section.
+
+## Fields
+
+We use Fields in Craft to define the different content types that are stored in an Entry. Craft provides some fields automatically (title, slug, postDate and more) but we can also define our own fields so we can mold Craft _around_ our content.
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Should the built-in fields be title case?

> (Title, Slug, Post Date, and more)

File: en/the-pieces-of-craft.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Let's say we are working on our cocktail lounge website and want to build out the Recipes section for the house drinks. 
+
+We've already created the Channel Section to hold our entries. Now we need to create the fields to hold our content within that section.
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Probably remove “Channel” here

File: en/the-pieces-of-craft.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+* Recipe Image
+* Recipe Snapshot (short description)
+* Recipe Content (ingredients and instructions)
+
+We could get even more detailed with the Recipe Content but this will be fine for now.
+
+
+## Templates
+
+We use Templates in Craft to output our website design and content. You can think of a template a HTML document.
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Think we’re missing a “like” here before “a HTML document”

File: en/the-pieces-of-craft.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Craft is agnostic to your website's layout. It doesn't use themes or impose any content layout or structure. You can use your own HTML/CSS/JS layout. By using
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Rogue “By using”

File: en/the-pieces-of-craft.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+## Templates
+
+We use Templates in Craft to output our website design and content. You can think of a template a HTML document.
+
+Craft is agnostic to your website's layout. It doesn't use themes or impose any content layout or structure. You can use your own HTML/CSS/JS layout. By using 
+
+Craft uses the [Twig templating engine](https://twig.symfony.com) to make it really easy for you to bring the website templates to life using the data you’ve saved in Craft.
+
+Twig is fantastic because it is flexible and fast. The twig website has a solid overview of what Twig is and how you can use it in your HTML templates.
```
Comment (brandonkelly) [inline en/the-pieces-of-craft.md]: Capitalize “twig”

File: en/the-pieces-of-craft.md (hunk 2 above)
Comment (ryanirelan) [reply en/the-pieces-of-craft.md]: Yes, let's make that the house style, along with Control Panel (from your previous comment).
